{"name": "Invalid value for param - Patches in the Request Body and getting Incorrect output -- @1.2 ", "status": "failed", "statusDetails": {"message": "AssertionError: failed with exception: Actual Output is: {'coords': [1, 2], 'patches': 2}\n", "trace": "  File \"c:\\users\\bansaan\\pycharmprojects\\platformscience\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\bansaan\\pycharmprojects\\platformscience\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\pltscisdetstepsimpl.py\", line 78, in verify_response_code_error\n    raise AssertionError(\"failed with exception: \" + str(e))\n"}, "steps": [{"name": "Given I Set POST request api endpoint", "status": "passed", "start": 1628433427333, "stop": 1628433427334}, {"name": "When I Set HEADER param request content type as \"application/json.\"", "status": "passed", "start": 1628433427334, "stop": 1628433427335}, {"name": "When Set invalid request Body { \"roomSize\" : [5,5], \"coords\" : [1, 2], \"patches\" : [[7, 7]], \"instructions\" : \"NEWSNEWS\" }", "status": "passed", "start": 1628433427335, "stop": 1628433427336}, {"name": "And Send a POST HTTP request", "status": "passed", "start": 1628433427337, "stop": 1628433427353}, {"name": "Then I receive error 500, \"Internal Server Error\"", "status": "failed", "statusDetails": {"message": "AssertionError: failed with exception: Actual Output is: {'coords': [1, 2], 'patches': 2}\n", "trace": "  File \"c:\\users\\bansaan\\pycharmprojects\\platformscience\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\bansaan\\pycharmprojects\\platformscience\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\pltscisdetstepsimpl.py\", line 78, in verify_response_code_error\n    raise AssertionError(\"failed with exception: \" + str(e))\n"}, "start": 1628433427353, "stop": 1628433427355}], "parameters": [{"name": "request_body", "value": "{ \"roomSize\" : [5,5], \"coords\" : [1, 2], \"patches\" : [[7, 7]], \"instructions\" : \"NEWSNEWS\" }"}, {"name": "response_code", "value": "500"}, {"name": "response_error", "value": "Internal Server Error"}], "start": 1628433427332, "stop": 1628433427356, "uuid": "e1f6d630-a0dd-4a1d-b9cf-617d95537a7e", "historyId": "11f7c4e8166fa7ccaba3fb3aaf2e469e", "fullName": "Test Imaginary Robot for Valid and Invalid Input values for Patches params: Invalid value for param - Patches in the Request Body and getting Incorrect output", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Test Imaginary Robot for Valid and Invalid Input values for Patches params"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}